### PR TITLE
Add notes about running the pipeline for large inputs.

### PR DESCRIPTION
**Pull request by arostamianfar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +76,21 @@ python -m gcp_variant_transforms.vcf_to_bq \
   --runner DataflowRunner
 ```
 
+#### Running with large inputs
+
+You may adjust the number of workers that Dataflow uses in the pipeline by
+adding the `--max_num_workers` argument. To request a specific number of
+workers, use the `--num_workers` argument. You may need to request additional
+quota for using large number of workers. Please see
```
Comment (mhsaul) [inline README.md]: nit: using *a* large

File: README.md [hunk 2]
```diff
@@ -72,6 +76,21 @@ python -m gcp_variant_transforms.vcf_to_bq \
...
+https://cloud.google.com/compute/quotas for details.
+
+If you want to load large number of files (e.g. more than 50), then you
```
Comment (mhsaul) [inline README.md]: nit: load *a* large

File: README.md [hunk 3]
```diff
@@ -72,6 +76,21 @@ python -m gcp_variant_transforms.vcf_to_bq \
...
+should specify `--representative_header_file` to point to a file with
```
Comment (mhsaul) [inline README.md]: s/with

File: README.md (hunk 1 above)
Comment (arostamianfar) [reply README.md]: Done. Thanks!

File: README.md (hunk 2 above)
Comment (arostamianfar) [reply README.md]: Done. Thanks!

File: README.md (hunk 3 above)
Comment (arostamianfar) [reply README.md]: Done. Thanks!

**Pull request merged**